.ci/scripts/install-gradle-profiler.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+PROFILER_VERSION="0.16.0"
+wget https://repo.gradle.org/gradle/ext-releases-local/org/gradle/profiler/gradle-profiler/$PROFILER_VERSION/gradle-profiler-$PROFILER_VERSION.zip -O $WORKSPACE/gradle-profiler-$PROFILER_VERSION.zip
+unzip $WORKSPACE/gradle-profiler-$PROFILER_VERSION.zip
+mv $WORKSPACE/gradle-profiler-$PROFILER_VERSION $WORKSPACE/gradle-profiler
+set -e
+$GRADLEW -S --max-workers=$MAX_WORKERS $@
```

Comment:
I think we should efer to the `run-gradle.sh` script here given it's what does stuff like set MAX_WORKERS.

Reply:
this changed. the gradle profiler doesn't use GRADLEW here as defined to run gradle.